reflex/.templates/jinja/web/utils/context.js.jinja2

```diff
@@ -23,6 +23,8 @@ export const clientStorage = {{ client_storage|json_dumps }}
 export const clientStorage = {}
 {% endif %}
 
+export const main_state_name = "{{const.main_state_name}}"
```

Comment:
I think this is already coming in as `state_name`.

reflex/compiler/compiler.py:90

Reply:
iirc this was the state_name for `rx.app.state`. will check

Reply:
Yes, they are not the same. `compile_contexts` gets called from `reflex.app._compile` with `self.state`

Reply:
maybe for clarity, we should call the var `root_state_name` instead of `main_state_name` just to indicate that it's the name of `rx.State`, the root of the state "tree"

Reply:
What about `main_state_dispatch` should we rename that one to `root_state_dispatch` as well?